notebooks/optimal-stopping-point/osp_model.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>Maybe this is beyond the scope of this notebook, but is there a way to "sanity check" this result or validate this result?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/445/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>This is something we will need to look into further and is also related to the issue we have created here: https://github.com/aicoe-aiops/ocp-ci-analysis/issues/365</p>